app/client/cli/keys.go

```diff
@@ -292,6 +270,9 @@ func keysGetCommands() []*cobra.Command {
 			},
 		},
 	}
+
+	applySubcommandOptions(cmds, attachKeybaseFlagsToSubcommands())
```

Comment:
i refactored it a bit to make it more clear which subcommands need options, I'm pretty sure this was needed since the keysGetCommand did not have any flags before this PR.

Reply:
@0xBigBoss Does the get command need any flags? Personally I think the `applySubcommandOptions` should be as the top to be clear which flags are being added to which subcommand

Reply:
`keysGetCommand` needs to know which keybase to get the key from.

I disagree, the options are tightly coupled with the commands and should be as close as possible to their definition.

Reply:
@0xBigBoss Just to give some context I also thought this when first integrating the keybase with the CLI but @Olshansk pointed out to me that its more for the other devs to pick up quickly instead of having to dive deep into the code to find out which cmds have which flags

https://github.com/pokt-network/pocket/pull/501#discussion_r1105082303

Reply:
Yah i get that it makes it easier to read, but we should be leveraging go compiler and IDE "lookup references" since these flags have bled into other files.

Plus I think it makes testing even more painful. Testing cobra cli apps is already kind of a dumpster fire though it is getting better. So I'm gonna say it's better to wrap it in a function that sets up the commands with all required inputs.

 That way tests like this, https://github.com/pokt-network/pocket/pull/537/files#diff-f16936f439473efd656c1cf821fa092b9eeb465ee369b8267c9c470832b2e2f4, are easier to write in the future.

Reply:
definitely a lot of this feedback on improving the DX of the CLI app should be collected and tackled in a separate PR IMO.